DOL.WHD.Section14c.Web/src/modules/components/sectionWorkSites/sectionWorkSitesController.js

```diff
@@ -131,6 +131,12 @@ module.exports = function(ngModule) {
             }
         }
 
+        this.siteRowClicked = function(e) {
+            let row = $(e.target).closest('.expanding-row');
+            row.toggleClass("expanded");
+            row.next().toggleClass("show");
+        }
```

Comment:
This makes me a little nervous because it requires the controller to have knowledge about the DOM structure.  If the markup changes, this could break.  I'm conflicted on it, though, because the alternative likely creates more Angular watches and those things have a tendency to lead to performance nightmares.